.rubocop.yml

```diff
@@ -26,6 +26,9 @@ Metrics/CyclomaticComplexity:
 Metrics/ParameterLists:
   Max: 10
 
+Style/AccessorMethodName:
+  Enabled: false
+
```

Comment:
Because of the `get_value` method: https://github.com/thread-pond/scrum-lint/blob/master/lib/scrum_lint/reporters/interactive_reporter.rb#L10